wpimath/util/units.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+
+from math import degrees, radians, pi
+
+kInchesPerFoot = 12.0
+kMetersPerInch = 0.0254
+kSecondsPerMinute = 60
+kMillisecondsPerSecond = 1000
+kKilogramsPerLb = 0.453592
+
+
+def metersToFeet(meters: float) -> float:
+    """Converts given meters to feet.
+
+    :param meters: The meters to convert to feet.
+
+    :returns: Feet converted from meters.
+    """
+    return metersToInches(meters) / kInchesPerFoot
+
+
+def feetToMeters(feet: float) -> float:
+    """Converts given feet to meters.
+
+    :param feet: The feet to convert to meters.
+
+    :returns: Meters converted from feet.
+    """
+    return inchesToMeters(feet * kInchesPerFoot)
+
+
+def metersToInches(meters: float) -> float:
+    """Converts given meters to inches.
+
+    :param meters: The meters to convert to inches.
+
+    :returns: Inches converted from meters.
+    """
+    return meters / kMetersPerInch
+
+
+def inchesToMeters(inches: float) -> float:
+    """Converts given inches to meters.
+
+    :param inches: The inches to convert to meters.
+
+    :returns: Meters converted from inches.
+    """
+    return inches * kMetersPerInch
+
+
+def degreesToRadians(degrees: float) -> float:
+    """Converts given degrees to radians.
+
+    :param degrees: The degrees to convert to radians.
+
+    :returns: Radians converted from degrees.
+    """
+    return radians(degrees)
+
+
+def radiansToDegrees(radians: float) -> float:
```

Comment:
Given these functions already exist in `math` I'm not sure having them here as well it's particularly useful.